### PR TITLE
parse multiple pages of gitlab repos

### PR DESCRIPTION
**Pull request by mastertinner**
Please review and merge or provide me with feedback

### CODE REVIEW

File: lib/api.js [hunk 1]
```diff
@@ -53,7 +54,44 @@ function get(config, uri, done) {
...
+              });
+
+            i++;
+            },
+            function(err, n) {
+              if(err) return done(err);
+
+              return done(null, results, res);
+            }
+          );
```
Comment (knownasilya) [inline lib/api.js]: 85-91 should probably be un-indented one time, otherwise everything looks good :+1:

**Comment by knownasilya** [comment]
How come you keep closing the PRs?

**Comment by mastertinner** [comment]
Sorry, I didn't know it was possible to modify a PR after it was done, that's why I created new ones for each change...

**Comment by knownasilya** [comment]
@mastertinner next time create a branch for your PR, and then you can keep commiting to that branch till it has been merged to update the PR.

Use a new branch for each PR you do.

**Comment by mastertinner** [comment]
Will do, thanks for letting me know!

**Pull request closed without merging**